email_marketing/manage_unsubscriptions.rst

```diff
@@ -0,0 +1,48 @@
+====================================
+Manage Unsubscriptions and Blacklist
+====================================
+It is best practice, and legally required, to allow recipients to unsubscribe from mailing lists
+as you do not want your audience to think that your company is using any tactics that are dishonest
+or spammy.
+
+Enable the Blacklist feature
+============================
+Go to :menuselection:`Configuration --> Settings` and enable the option *Blacklist Option when
+unsubscribing*.
+
+
+.. image:: media/manage1.png
+   :align: center
+   :alt: Blacklist in Odoo Email Marketing
+
+
+Now, once the user clicks on the *Unsubscribe* link on your email, he will be redirected to the
+following page:
+
+
+.. image:: media/manage2.png
```

Comment:
This image looks quite massive imho. You could maybe try to play with the subdirectives of `image` to adjust the scaling ?